src/openrct2/world/SmallScenery.h

```diff
@@ -11,5 +11,8 @@
 
 enum
 {
+    // Pre Extended Colour set: Support flag stored in bit 5 of primary colour
+    MAP_ELEM_SMALL_SCENERY_COLOUR_FLAG_NEEDS_SUPPORTS_LEGACY = (1 << 5),
```

Comment:
This is a rct12 only flag so should be in the RCT12 headers. Have a look at `RCT12_TILE_ELEMENT_LARGE_TYPE_MASK`

Reply:
.park files prior to this PR's version also use the support flag in the primary color. I've modified the conversion in parkfile.cpp to use this constant for clarity.

Reply:
It should still not be in this file. Create a duplicate of it in RCT12 headers and use it for all the RCT12 functions. I'd prefer if you move this one into the park file as a constant only used there. People shouldn't have to know anything about the legacy setup.

Reply:
Ok, that makes sense. Will do.